src/components/CharacterAssignment.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from 'react';
+import { DragDropContext } from 'react-beautiful-dnd';
+import ActorColumn from './ActorColumn.jsx'
+import UnassignedColumn from './UnassignedColumn';
+
+// Entire Drag and Drop context for assigning characters to actors
+const CharacterAssignment = ({actors, currentScript}) => {
+  const [columnOrder, setColumnOrder] = useState([])
```

Comment:
`columnOrder` seems to imply some sort of configuration for _what_ order a column is in, such as "alphabetical ascending", "alphabetical descending", "lines ascending", etc.
From the rest of the code, it's clear this is not the case, but I'm having trouble understanding what this is actually for.

Reply:
These variables may warrant a comment explaining their purpose if a clear enough name isn't feasible.

Reply:
Good catch. The naming was initially established with the expectation of the user being able to rearrange the columns themselves. We decided that wasn't a necessary functionality for our use case, so I removed the functionality but didn't think to rename the bit of state. I do believe it's still a helpful piece of state to have as a more concise representation of the actor columns to be able to iterate through when rendering the columns.